recipes/tket/conanfile.py

```diff
@@ -106,29 +102,9 @@ def configure(self):
         self.options["eigen"].MPL2_only = True
 
     def build(self):
-        # Build with boost patches
-        boost_include_path = self.deps_cpp_info["boost"].include_paths[0]
-        curdir = os.path.dirname(os.path.realpath(__file__))
-        patches = {
-            # Patch pending merge of https://github.com/boostorg/graph/pull/280
-            # and new boost release. (Note that PR implements a different solution so
-            # code will need updating as well.)
-            os.path.join(
-                boost_include_path, "boost", "graph", "vf2_sub_graph_iso.hpp"
-            ): os.path.join(curdir, "patches", "vf2_sub_graph_iso.diff"),
-        }
-        for filepath, patch_file in patches.items():
-            print("Patching " + filepath)
-            shutil.copyfile(filepath, filepath + ".original")
-            tools.patch(base_path=boost_include_path, patch_file=patch_file)
         cmake = self._configure_cmake()
-        try:
-            print("Building")
-            cmake.build()
-        finally:
-            for filepath in patches.keys():
-                print("Restoring " + filepath)
-                shutil.move(filepath + ".original", filepath)
+        print("Building")
```

Comment:
I'd delete this print line as well as it isn't useful now.

Reply:
done